samples/client/petstore/qt5cpp/client/SWGUserApi.h

```diff
@@ -35,14 +35,14 @@ class SWGUserApi: public QObject {
     QString basePath;
     QMap<QString, QString> defaultHeaders;
 
-    void createUser(SWGUser body);
-    void createUsersWithArrayInput(QList<SWGUser*>* body);
-    void createUsersWithListInput(QList<SWGUser*>* body);
+    void createUser(SWGUser& body);
+    void createUsersWithArrayInput(QList<SWGUser*>*& body);
+    void createUsersWithListInput(QList<SWGUser*>*& body);
     void deleteUser(QString* username);
     void getUserByName(QString* username);
     void loginUser(QString* username, QString* password);
     void logoutUser();
-    void updateUser(QString* username, SWGUser body);
+    void updateUser(QString* username, SWGUser& body);
```

Comment:
Did you make this parameter a reference to avoid copies, or to be able to modify it inside the functions? If it's to avoid copies, please make it a `const` reference.
(same remark for createUser() above)

Reply:
Yes . a const reference would be better. I will check it.